cmd/kops/create_ig.go

```diff
@@ -96,6 +98,7 @@ func NewCmdCreateInstanceGroup(f *util.Factory, out io.Writer) *cobra.Command {
 	// DryRun mode that will print YAML or JSON
 	cmd.Flags().BoolVar(&options.DryRun, "dry-run", options.DryRun, "If true, only print the object that would be sent, without sending it. This flag can be used to create a cluster YAML or JSON manifest.")
 	cmd.Flags().StringVarP(&options.Output, "output", "o", options.Output, "Ouput format. One of json|yaml")
+	cmd.Flags().BoolVar(&options.NoEditor, "no-editor", options.NoEditor, "If true, The instance group will be created by default with no option of editing it")
```

Comment:
I know this is nit-picking - sorry -  but how about `--edit` with a default of true?  I personally don't find negative flags not hard to think about ;-)

Otherwise I think this is great :-)

Reply:
I did this in oder not to break how this was done before. If you want me to change it to be more understandable or to follow how the rest of the commands work, I'll do it.

Reply:
I _think_ if we set the default to true (the third arg is the default) we can have `edit` with a default of true (?)